test/conformance/conformance_test.go

```diff
@@ -72,10 +164,10 @@ func TestGatewayConformance(t *testing.T) {
 	opts.Implementation = conformancev1.Implementation{
 		Organization: metadata.Organization,
 		Project:      metadata.ProjectName,
-		URL:          metadata.ProjectURL,
+		URL:          metadata.RepoURL,
 		Version:      metadata.Release,
 		Contact: []string{
-			path.Join(metadata.ProjectURL, "/issues/new/choose"),
+			path.Join(metadata.RepoURL, "/issues/new/choose"),
```

Comment:
```suggestion
			metadata.RepoURL + "/issues/new/choose",
```
Otherwise with the change in `Makefile` to include `https://github...` the double slash will be reduced to a single one by `path.Join` because it's for filepaths and not URIs (btw. Golang does not have a handy function for joining such paths)